device/common/include/traccc/device/fill_prefix_sum.hpp

```diff
@@ -0,0 +1,39 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
```

Comment:
I was not paying attention to this so far. But on any new file you should just use the current year. We try to show in these file headers when the code was actually developed.

Of course let's not worry about this for any of the other files. But here we may as well set it just to `2022`.